src/enc_jpeg2000.c

```diff
@@ -56,9 +56,9 @@
  * 1 = try increasing number of guard bits otherwise, no additional options
  * @param[in] jpclen  - Number of bytes allocated for new JPEG2000 code stream in outjpc.
  * @param[in] outjpc - Output encoded JPEG2000 code stream
- * @return 0 successfully decode JPEG2000 code stream
- *         -  -3 Error decode jpeg2000 code stream.
- *         -  -5 decoded image had multiple color components.
+ * @return 0 = Length in bytes of encoded JPEG2000 code stream
```

Comment:
I have been taking out = in this sort of text. So return it to the way it was before, without the = sign.